src/SFA.DAS.EAS.Web/PublicHashingService/PublicHashingService.cs

```diff
@@ -0,0 +1,10 @@
+namespace SFA.DAS.EmployerCommitments.Web.PublicHashingService
+{
+    public class PublicHashingService : HashingService.HashingService, IPublicHashingService
```

Comment:
This class is not necessary.

The existing HashingService could also implement IPublicHashingService, and DefaultRegistry could use...

For<IPublicHashingService>().Use(x => new HashingService.HashingService(config.PublicAllowedHashstringCharacters, config.PublicHashstring));

(You could still implement a PublicHashingService at a later point that implemented IPublicHashingService, if you e.g. wanted to change the implementation, without having to change any consuming code.)

Reply:
not possible, as HashingService is in internal librray